DATA-7264: Document virtual tree behavior for categories (#353)
AdobeDocs/commerce-services: src/pages/optimizer/merchandising-services/categories-storefront-implementation.md

See the [CategoryTree query examples](#categorytree-query-examples) and [searchCategory query examples](#searchcategory-query-examples) sections for example queries and responses using this type.

## How category families work

> A **family** is a named tag you apply to category nodes in the canonical category tree. Use families to scope navigation menus, category trees, and search results to a subset of your catalog—for example, a seasonal promotion or a specific storefront—without duplicating categories.

Reviewer: Break the second sentence up without using the dashes

> A **family** is a named tag you apply to category nodes in the canonical category tree. Use families to scope navigation menus, category trees, and search results to a subset of your catalog—for example, a seasonal promotion or a specific storefront—without duplicating categories.

Author: ```suggestion
A **family** is a named tag you apply to category nodes in the canonical category tree. Use families to scope navigation menus, category trees, and search results to a subset of your catalog, such as a seasonal promotion or a specific storefront. This way, you avoid duplicating categories.
```

### Optional fields add overhead

> The `description`, `metaTags`, and `images` fields on `categoryTree` are optional. Selecting them adds negligible latency—at most about 7 ms in testing. Exclude these fields when a view doesn't need descriptive content or SEO metadata. Excluding them is about payload relevance, not performance.

Reviewer: Are we supposed to avoid contractions?

Another em dash phrase I don't like, but I won't insist on changing.

> The `description`, `metaTags`, and `images` fields on `categoryTree` are optional. Selecting them adds negligible latency—at most about 7 ms in testing. Exclude these fields when a view doesn't need descriptive content or SEO metadata. Excluding them is about payload relevance, not performance.

Author: ```suggestion
The `description`, `metaTags`, and `images` fields on `categoryTree` are optional. Selecting them adds negligible latency—at most about 7 ms in testing. Exclude these fields when a view does not need descriptive content or SEO metadata. Excluding them is about payload relevance, not performance.
```
